spec/serializers/rest/status_serializer_spec.rb

```diff
@@ -51,5 +51,17 @@
         expect(subject['favourites_count']).to eq(40)
       end
     end
+
+    context 'with created_at' do
+      it 'is serialized as RFC 3339 datetime' do
+        expect { DateTime.rfc3339(subject['created_at']) }.to_not raise_error
+      end
+    end
+
+    context 'when edited_at is populated', pending: 'How to mark status as edited?' do
```

Comment:
There's an `UpdateStatusService` which is actually used to do this ... but you might be able to get away with just `Fabricate.build :status, edited_at: 3.days.ago` or something for purposes of this serialization spec.